recipes/textattack/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "textattack" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/QData/{{ name|title }}/archive/{{ version }}.tar.gz
+  sha256: 9ccb8b2a43120ee2d140fda78d4dfd7f0339e6d051a2c67cc9e891df2b993d85
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [not linux or py<37]
```

Comment:
Same, noarch

Reply:
Should I keep the skip? I'm still fuzzy on how exactly noarch and skips work together.